core/arch/x86/decode.c

```diff
@@ -2009,6 +2009,7 @@ decode_operand(decode_info_t *di, byte optype, opnd_size_t opsize, opnd_t *opnd)
          */
```

Comment:
> Add instructions vunpcklps, vunpcklpd, more see below.

There must be something which makes all these instructions similar: why did you group these particular ones?  Use that in the title line.  E.g., "Add evex-encoded 0f{1,2}* opcodes" or whatever it is.

Reply:
There was no reason except they are taken from the big list in i#1312. I am going down the list. There **are** however all evex-promoted opcodes, e.g. ones that already exist in AVX. That's the only grouping that had been done in this regard.

Reply:
Ok initially, most of them had xmm-only operands, I'll take this as the title. It's not entirely true for all instructions added.